.jules/runs/bolt_analysis_stack_builder/pr_body.md

```
@@ -1,48 +1,57 @@
 ## 💡 Summary
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)